_posts/2020-03-06-introducing-mirrormaker2.md

```diff
@@ -0,0 +1,268 @@
+Typical scenarios where you might consider MirrorMaker are for disaster recovery and increased throughput.
+
+With the release of Strimzi 0.17, things become a little more interesting with the introduction of support for MirrorMaker 2.0.
+MirrorMaker 2.0 represents a significant shift in the way you synchronize data between replicated Kafka clusters,
+promising a more dynamic and automated approach to topic replication between clusters.
+
+<!--more-->
+
+### MirrorMaker 2.0 - the Kafka Connect(ion)
+
+The previous version of MirrorMaker relies on configuration of a _source producer_ and _target consumer_ pair to synchronize Kafka clusters.
+
+MirrorMaker 2.0 is based on Kafka Connect, which is a bit of a game changer.
+For a start, there is no longer a need to configure producers and consumers to make your connection.
+
+With MirrorMaker 2.0 deployed with Strimzi, you identify your source and target clusters.
+You then configure and deploy MirrorMaker 2.0 to make the connection between those clusters.
+
+This image shows a single source cluster, but you can have multiple source clusters.
+
+![Connecting clusters with MirrorMaker 2.0](/assets/2020-03-12-mirrormaker.png)
+
+MirrorMaker 2.0 **_connectors_** -- remember, we're based on Kafka Connect now -- and related **_internal topics_** help manage the transfer and synchronization of data between the clusters.
+
+Different to the previous version of MirrorMaker, radically different, but different doesn't mean more complicated here.
+In fact, once you know the essentials, setting up is rather straightforward.
+
+Using Strimzi, you just need to work out how you want to configure a `KafkaMirrorMaker2` resource, which will define your Kafka Connect deployment and start running a set of MirrorMaker 2.0 connectors.
+We'll look a bit more at configuring the `KafkaMirrorMaker2` resource later on.
+
+First, something you'll be keen to know.
+Is the sequel better than the original?
+Well, how does **_bidirectional replication_** sound?
+And what about **_topic configuration synchronization_**  and **_offset tracking_**?
+Pretty good, right?
+
+This is MirrorMaker unleashed, ready to fulfill the potential we always knew it had.
+
+### Bidirectional opportunities
+
+In the previous version of MirrorMaker, the topic name in the source cluster is automatically created in the downstream cluster.
+Fine, to a point.
+But there is no distinction between _Topic-1.Partition-1_ in the source cluster and _Topic-1.Partition-1_ in the target cluster.
+Essentially, you are limited to _active/passive_ synchronization.
+MirrorMaker 2.0 solves this by introducing the concept of _remote_ topics.
+
+_Remote_ topics are created from the originating cluster by the `MirrorSourceConnector`.
+They are distinguished by automatic renaming that appends the name of cluster to the name of the topic.
+Our _Topic-1.Partition-1_ from the source cluster becomes the never-to-be-confused _Cluster-1-Topic-1.Partition-1_ in the target cluster.
```

Comment:
it _Cluster-1.Topic-1.Partition-1_ (see the "." after the cluster name instead of the "-")

Reply:
We use _Cluster-1-_ in the image too